docs/source/guide/labeling.md

```diff
@@ -137,13 +137,33 @@ To label tasks multiple times while the minimum annotations required is set to o
 
 Bulk labeling is useful when you have a large set of tasks that share a common label or annotation that you want to apply in one go. For example:
 
-* **Image classification**: Quickly label multiple images with the same category (e.g., “Cat” vs. “Dog”). 
+* **Data classification**: Quickly label multiple tasks with the same category. 
 * **Filtering and grouping**: Use filters to isolate tasks matching specific criteria (e.g., tasks you know are all “Approved”), then label them in bulk. 
 * **Speed and consistency**: Avoid repetitive labeling for items that can confidently share a label.
 
+#### Supported data types
+
+Bulk labeling supports all data types (e.g. images, text, audio, video). 
+
+#### Supported labeling tasks
+
+Bulk labeling supports classification tasks and text area input. You can use any combination of supported tags. 
+
+The following control tags can be assigned through the bulk labeling action:
+
+* [Choices](/tags/choices.html)
+* [DateTime](/tags/datetime.html)
+* [Number](/tags/number.html)
+* [Rating](/tags/rating.html)
+* [Taxonomy](/tags/taxonomy.html)
+* [TextArea](/tags/textarea.html)
+
+If your labeling configuration includes tags not listed above, the **Bulk labeling** action is not accessible. 
```

Comment:
Other tags are supported, just not other control tags, so like Labels are not supported, but Image definitely is.